kubectl-cloudflow/verify/util.go

```diff
@@ -0,0 +1,169 @@
+package verify
```

Comment:
Was a bit confused by util.go being part of package verify when looking at the use of the package private function `mkString` in verifyBlueprint

Reply:
agreed. Can we have this `util` as a separate package ? Maybe with a less kitchensink name than util ..

Reply:
ok sure.